docker-compose.yml-example

```diff
@@ -55,6 +55,43 @@ services:
       - ldap-net
       - web-net
 
+  password-self-service:
+    image: tiredofit/self-service-password
+    container_name: password-self-service
+    domainname: ${LDAP_DOMAIN}
+    depends_on:
+      - ldap
+      - mail
+    environment:
+      - LDAP_SERVER=ldap://ldap:389
+      - LDAP_BINDDN=cn=admin,dc=kopano,dc=demo
+      - LDAP_BINDPASS=${LDAP_ADMIN_PASSWORD}
+      - LDAP_BASE_SEARCH=${LDAP_BASE_DN}
+      - MAIL_FROM=noreply@${LDAP_DOMAIN}
+      - SMTP_HOST=mail
+      - SMTP_PORT=25
+      - SMTP_SECURE_TYPE=false
+      - SMTP_AUTOTLS=false
+      - QUESTIONS_ENABLED=false
+      - PASSWORD_NO_REUSE=true
+      - WHO_CAN_CHANGE_PASSWORD=user
+      - SECRETEKEY=${SELF_SERVICE_SECRETEKEY}
+      - BACKGROUND=.
+      - PASSWORD_MIN_LENGTH=${SELF_SERVICE_PASSWORD_MIN_LENGTH}
+      - PASSWORD_MAX_LENGTH=${SELF_SERVICE_PASSWORD_MAX_LENGTH}
+      - PASSWORD_MIN_LOWERCASE=${SELF_SERVICE_PASSWORD_MIN_LOWERCASE}
+      - PASSWORD_MIN_UPPERCASE=${SELF_SERVICE_PASSWORD_MIN_UPPERCASE}
+      - PASSWORD_MIN_DIGIT=${SELF_SERVICE_PASSWORD_MIN_DIGIT}
+      - PASSWORD_MIN_SPECIAL=${SELF_SERVICE_PASSWORD_MIN_SPECIAL}
+    expose:
+      - "80"
+    volumes:
+      - password-self-service:/www/ssp
```

Comment:
in relation to the comment with doing a git checkout of ssp instead of using a release. having this as a proper volume means that users would need to enter into the running container and doing a `git pull` to get a newer ssp version. if this volume is removed it would download a new release once the container is recreated locally.

Reply:
I started a discussion to solve this at the used docker image project:
https://github.com/tiredofit/docker-self-service-password/issues/10

Reply:
to have it said here as well, afais just remove the volumes from this image